manifests/agent.pp

```diff
@@ -253,6 +256,7 @@
   $zbx_group_create                               = $zabbix::params::agent_zbx_group_create,
   $zbx_templates                                  = $zabbix::params::agent_zbx_templates,
   Array[Hash] $zbx_macros                         = [],
+  Integer $zbx_interface_type                     = 1,
```

Comment:
can you enforce the datatype a bit more? are negative values allowed? I think there is an upper limit that could be enforced as well?